### PR TITLE
Discussion thread links

### PR DESCRIPTION
**Pull request by nickoferrall**
This PR resolves issue #4345.

### Consideration

Copying and pasting between Draft JS editors doesn't work very well - font styles and hyperlinks aren't always maintained. This is a [known issue](https://github.com/facebook/draft-js/issues/787) in Draft JS. Someone from the community has created a PR with a suggested solution that hasn't been approved but I'd be happy to look into it. 

However, I can't imagine that many of our users are copying & pasting text that they've just pasted so I suggest that I create an issue for this and prioritise it if any user tension pops up. 

### AC

- [ ] Users can paste links into the comment editor
- [ ] Users can paste links into the task editor
- [ ] Users can paste a combination of text and hyperlinks and the hyperlinks will still work

### CODE REVIEW

File: packages/client/components/TaskEditor/CommentEditor.tsx [hunk 1]
```diff
@@ -164,6 +166,15 @@ const CommentEditor = (props: Props) => {
         }
       }
     }
+    const links = linkify.match(text)
```
Comment (nickoferrall) [inline packages/client/components/TaskEditor/CommentEditor.tsx:169]: Initially I included this code in `useLinks` to keep things DRY but as it's only being repeated twice and as we need the above `onPasteText` code with the `ref` in this file, I felt it'd be best to keep it in the same function

**Comment by mattkrick** [comment]
this isn't working for me:
1. in a thread, add a comment & type `http://www.google.com`
2. hit space, see it turn into a hyperlink, copy the text
3. submit
4. hit ctrl+v to add a second, duplicate comment
5. clicking the first comment works as a hyperlink, but the 2nd does not

**Comment by nickoferrall** [comment]
Interesting - I couldn't replicate that behaviour in the comment editor but if I followed steps 1-3 and then did ctrl+v into the task editor, the link was underlined but didn't work. 

I'm now trimming the whitespace which has resolved the task editor issue and I think it should resolve the comment editor issue too.

**Comment by mattkrick** [comment]
success!

**Pull request merged**